plenary/plenary-25.yaml

```diff
@@ -0,0 +1,340 @@
+    title: "Information on ISO ballots and reviews"
+    identifier: 279
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 2008-11-25
+          end: 2008-11-26
+        message: ISO/TC 154 resolves to request ISO/CS and the secretary of ISO/TC 154 to be informed on ballots and reviews that will and have been launched via the ISO balloting system
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Confirmation of ISO 7372:2005"
+    identifier: 279
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 2008-11-25
+          end: 2008-11-26
+        message: ISO/TC 154, having considered the result of the systematic review of ISO 7372:2005 Trade data interchange – Trade data elements directory (document 154n506), resolves to confirm ISO 7372:2005
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Way forward on TDED"
+    identifier: 280
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 2008-11-25
+          end: 2008-11-26
+        message: ISO/TC 154 resolves to launch a joint study with UN/CEFACT to determine an integrated strategy for the development of a single library including at least TDED, EDED and CCL
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Invitation to ISO 20022 RA to participate in discussions of TDED Joint Maintenance Agency"
+    identifier: 281
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: encourages
+        date_effective:
+          start: 2008-11-25
+          end: 2008-11-26
+        message: ISO/TC 154 encourages ISO 20022 RA to participate actively in the work of the TDED joint MA
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Election of Ms. Mary Kay Blantz as co-chair of the TDED Joint Maintenance Agency"
+    identifier: 282
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: elects
+        date_effective:
+          start: 2008-11-25
+          end: 2008-11-26
+        message: ISO/TC 154 elects Ms. Mary Kay Blantz as co-chair of the TDED Joint Maintenance Agency
```

Comment:
Here, I used the word "elects" as an action type. Is that correct?

Original source reference:
![capture1](https://user-images.githubusercontent.com/33627611/71758634-e455c180-2e78-11ea-8bd3-77e51971c2b0.JPG)

Reply:
Agree.